File: Java/atv_2189.java
import java.util.Scanner;

public class atv_2189 {

    final static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {

        int teste = 0;
        
        while (true) {
            
            int participantes = input.nextInt();
            int ingresso []= new int[participantes];
            
            if (participantes==0){
                    break;
                }
            
            for(int i = 0; i<participantes; i++){
                ingresso[i]=input.nextInt();
            }
            
            for (int i=0; i<participantes; i++){
                if(ingresso[i]==(i + 1)){
                    teste++;
                    System.out.println("Teste "+teste);
                    System.out.println(i + 1);
                    System.out.println();
                    break;
                }
            }
        }
    }
}

File: Python/atv_2189.py
teste = 0

while True:
    participantes = int(input())
    
    if participantes == 0:
        break

    ingresso = [int(x) for x in input().split()]
    i = int()
    
    for i in range(participantes):
        if ingresso[i] == i + 1:
            teste += 1
            print("Teste", teste)
            print(i + 1)
            print()